.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: java
+install: mvn install -DskipTests -Dgpg.skip
```

Comment:
Lets add `-Dmaven.javadoc.skip=true -B -V` options here as well. This would bring the changes in par to what travis currently runs with only the addition of `-Dgpg.skip`.

`-Dmaven.javadoc.skip=true` - No need to build / install javadocs deps here
`-B` - Runs in batch mode rather than interactive. Also doesn't include a bunch of incremental install log lines making it easier to read
`-V` - Just shows the maven version info at the beginning of the command